packages/table/src/table.tsx

```diff
@@ -540,56 +526,28 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
                 ref={this.setRootTableRef}
                 onScroll={this.handleRootScroll}
             >
-                <TableQuadrant
+                <TableQuadrantStack
```

Comment:
Is there a better name than this? I can't think of one, but this is a little silly.